app/routes/task.py

```diff
@@ -0,0 +1,113 @@
+from flask import abort, Blueprint, jsonify, make_response, request
+from app import db
+from app.models.task import Task
+from datetime import datetime
+
+
+import requests
+import os
+
+task_bp = Blueprint("task", __name__, url_prefix="/tasks")
+
+
+@task_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    try:
+        new_task = Task(
+            title=request_body["title"],
+            description=request_body["description"]
+        )
+    except:
+        return abort(make_response({"details": "Invalid data"}, 400))
```

Comment:
Again using validation here is good, but indicating what's wrong would be better.